server/server.js

```diff
@@ -56,27 +56,75 @@ app.middleware('parse', bodyParser.urlencoded({
   extended: true
 }));
 
-// The access token is only available after boot
-app.middleware('auth', loopback.token({
-  model: app.models.accessToken,
-  currentUserLiteral: 'me'
-}));
 
-app.use(loopback.token());
+
+// Establish current user from access token
+app.use(loopback.context());
+app.use(function setCurrentUser(req, res, next) {
+    if (!req.accessToken) {
+        return next();
+    }
+    app.models.Person.findById(req.accessToken.userId, function(err, user) {
+        if (err) {
+            return next(err);
+        }
+        if (!user) {
+            return next(new Error('No user with this access token was found.'));
+        }
+
+        // NOTE: This section is pulled from passport-configurator.js
+        // sorry.
+        user.identities(function(err, identities) {
```

Comment:
Some of the views expect `.profiles` and `.accounts`. Probably not good dependencies to have, since they aren't technically part of the REST API. Anyway, this is a patch to ensure the views don't break.